hr_timesheet_purchase_order/models/hr_employee_base.py

```diff
@@ -8,3 +12,237 @@ class HrEmployeeBase(models.AbstractModel):
         string="Generate POs from Timesheet", default=False
     )
     billing_partner_id = fields.Many2one("res.partner")
+
+    is_auto_po_generate = fields.Boolean(
+        string="Automatic PO generation", default=False
```

Comment:
By default boolean field set "false" value.

Reply:
ok,I'll fix it